src/lib/statsig/statsig.tsx

```diff
@@ -78,6 +78,10 @@ function toStatsigUser(did: string | undefined) {
   return {
     userID,
     platform: Platform.OS,
+    custom: {
+      // Need to specify here too for gating.
+      platform: Platform.OS,
```

Comment:
Gating by platform didn't work unless I put it under `custom` namespace. Maybe we don't have to send both but let's keep both for now.